api/README.md

```diff
@@ -91,26 +92,85 @@ watch KUBECONFIG=/tmp/kubeconfig kubectl get pod -A
-knative-serving   activator-7fdbbcf6dc-jkcfn                1/1     Running   0          20m
-default           spark-spark-operator-59c685545-5mrtg      1/1     Running   0          20m
-istio-system      svclb-istio-ingressgateway-b9z82          4/4     Running   0          19m
-istio-system      svclb-istio-ingressgateway-n9zzk          4/4     Running   0          19m
-istio-system      svclb-istio-ingressgateway-77scg          4/4     Running   0          19m
-istio-system      svclb-istio-ingressgateway-l26k7          4/4     Running   0          19m
-istio-system      cluster-local-gateway-5bf54b4999-p2bv7    1/1     Running   0          19m
-istio-system      istio-ingressgateway-555bdcd566-xcn6h     1/1     Running   0          19m
+knative-serving   autoscaler-6f6f6cf579-czrmw               1/1     Running   0          17h
+knative-serving   controller-66f68465cd-d58mq               1/1     Running   0          17h
+knative-serving   domain-mapping-948df9f76-mwkhj            1/1     Running   0          17h
+knative-serving   webhook-7877b457fd-vhcvm                  1/1     Running   0          17h
+knative-serving   net-istio-webhook-695d588d65-r6szn        1/1     Running   0          17h
+knative-serving   autoscaler-hpa-66d97b559f-ts6zf           1/1     Running   0          17h
+knative-serving   net-istio-controller-544874485d-mncx9     1/1     Running   0          17h
+default           mockserver-7844d797f7-zwk2b               1/1     Running   0          17h
+kube-system       local-path-provisioner-84bb864455-58c7z   1/1     Running   0          17h
+kube-system       coredns-96cc4f57d-t8vsw                   1/1     Running   0          17h
+knative-serving   domainmapping-webhook-74fc9b87b4-pmqdg    1/1     Running   0          17h
+istio-system      istiod-7c595445b6-r6fwt                   1/1     Running   0          17h
+knative-serving   activator-7d658db58b-29pnb                1/1     Running   0          17h
+kube-system       metrics-server-ff9dbcb6c-bjwl2            1/1     Running   0          17h
+istio-system      svclb-istio-ingressgateway-kxtmp          3/3     Running   0          17h
+istio-system      svclb-istio-ingressgateway-xszc7          3/3     Running   0          17h
+istio-system      svclb-istio-ingressgateway-xg6zg          3/3     Running   0          17h
+default           spark-spark-operator-59c685545-qqn44      1/1     Running   0          17h
+istio-system      svclb-istio-ingressgateway-mn9n4          3/3     Running   0          17h
+istio-system      istio-ingressgateway-b7ffbd9c6-kfpl2      1/1     Running   0          17h
 ```
 
+#### <b>Prepare proprietary experiment engine</b>
+---
+Next, let's get the proprietary experiment engine plugin ready!
+
+1. Build the proprietary experiment engine plugin binary
+
+When building the binary, make sure you're using the same GOOS and GOARCH that's compatible with your machine. Eg. for MacOS M1 would be `GOOS=darwin GOARCH=arm64`.
```

Comment:
Rather than documenting the differences here, should we just make the `GOOS` property an ARG to the Dockerfile which is set to `linux` by default? Can the rest of the commands be used as is? Then, the same file could be used for both local and CI envs.